modules/nw-exposing-service.adoc

```diff
@@ -72,17 +72,41 @@ HTTP/1.1 200 OK
 
 endif::nodeport[]
 ifdef::nodeport[]
-. To expose a node port for the application, enter the following command. {product-title} automatically selects an available port in the `30000-32767` range.
+. To expose a node port for the application, modify the custom resource definition (CRD) of a service by entering by the following command:
 +
 [source,terminal]
 ----
-$ oc expose service nodejs-ex  --type=NodePort --name=nodejs-ex-nodeport --generator="service/v2"
+$ oc edit svc <service_name>
+----
++
+.Example output
+[source,yaml]
+----
+spec:
+  ports:
+  - name: 8443-tcp
+    nodePort: 30327 <1>
+    port: 8443
+    protocol: TCP
+    targetPort: 8443
+  sessionAffinity: None
+  type: NodePort <2>
+----
+<1> Optional: Specify the node port range for the application. {product-title} automatically selects an available port in the `30000-32767` range.
```

Comment:
```suggestion
<1> Optional: Specify the node port range for the application. By default, {product-title} selects an available port in the `30000-32767` range.
```

Reply:
And is a range something that we can insert into the YAML? If not, consider:

```suggestion
<1> Optional: Specify the node port range for the application. By default {product-title}, selects an available port in the 30000-32767 range.
```

Monospace on the dash implies that it is part of code or a configuration file.